contracts/test/TickTest.sol

```diff
@@ -26,4 +26,43 @@ contract TickTest {
     ) external view returns (uint256 feeGrowthInside0X128, uint256 feeGrowthInside1X128) {
         return ticks.getFeeGrowthInside(tickLower, tickUpper, tickCurrent, feeGrowthGlobal0X128, feeGrowthGlobal1X128);
     }
+
+    function update(
+        int24 tick,
+        int24 tickCurrent,
+        int128 liquidityDelta,
+        uint256 feeGrowthGlobal0X128,
+        uint256 feeGrowthGlobal1X128,
+        uint160 secondsPerLiquidityCumulativeX128,
+        uint32 time,
+        bool upper,
+        uint128 maxLiquidity
+    ) external returns (bool flipped) {
+        return
+            ticks.update(
+                tick,
+                tickCurrent,
+                liquidityDelta,
+                feeGrowthGlobal0X128,
+                feeGrowthGlobal1X128,
+                secondsPerLiquidityCumulativeX128,
+                time,
+                upper,
+                maxLiquidity
+            );
+    }
+
+    //    function clear(int24 tick) external {
```

Comment:
nit: unused

Reply:
yeah left it because we have skipped tests and it's only unused because echidna breaks

Reply:
added comment

Reply:
well echidna's broken anyway, might as well uncomment